hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -229,6 +246,42 @@ def apply_embedding_config(arg1, arg2, arg3, origin_call=None) -> int:
     return status_code
 
 
+def apply_reranker_config(
+    reranker_api_key: str | None = None,
```

Comment:
get error on py3.9
![image](https://github.com/user-attachments/assets/83e9158f-5b2d-43a5-b99f-3125ab604e23)

Reply:
> get error on py3.9 ![image](https://private-user-images.githubusercontent.com/58508660/364303500-83e9158f-5b2d-43a5-b99f-3125ab604e23.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU0NDA1NDgsIm5iZiI6MTcyNTQ0MDI0OCwicGF0aCI6Ii81ODUwODY2MC8zNjQzMDM1MDAtODNlOTE1OGYtNWIyZC00M2E1LWI5OWYtMzEyNWFiNjA0ZTIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA0VDA4NTcyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhYzMyN2ZhZGI5ZGUyM2IyMmVjMDYwMjJiNzE3MDYxMmNkYWI4MDI4Y2RjZjZmYTc3N2FjMGIzYzc2OGQ5OGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pM0Cu9eGAuiAOKg7QP1uPW7big0YoK5GV0qVJhCAoEg)

Does `Optional` fix it?

Reply:
yes, `Optional` would be fine